### ISSUE TITLE
New comment local entry for user satisfaction

### ISSUE THREAD

**Issue by noumantahir** [ecency/ecency-mobile#1923]
Just like upvotes, when user replies to a post it takes some time before that new comment appear in post response. it can usually take one minute giving user a worry regarding whether or not the reply actually got registered.

**To Reproduce**
1. Make a comment to any post
2. Refresh post details page right after
3. The comment would not appear

**Expected behavior**
The experience should feel as snappy and response as possible.

**Additional context**
Just like we manage local vote map, we can manage local comment map, though we will have to slightly more intelligent about discard the local comments on post refresh as user may refresh the post within one minute.

**Comment by feruzm** [comment]
Yes, let's do it... replicating website behaviour is best for consistent and pleasing user experience.

**Comment by feruzm** [comment]
Closing as implemented: replies are now optimistically inserted into the discussions query cache (including thread views), so a new comment appears immediately.
